osf/models/provider.py

```diff
@@ -181,9 +181,9 @@ def has_highlighted_subjects(self):
     @property
     def highlighted_subjects(self):
         if self.has_highlighted_subjects:
-            return self.subjects.filter(highlighted=True).order_by('text')[:10]
+            return self.subjects.filter(highlighted=True).order_by('text')
         else:
-            return sorted(self.top_level_subjects, key=lambda s: s.text)[:10]
```

Comment:
If there aren't highlighted subjects, I don't thiiiiink we want to return all of them. That would return the whole of the top-level of the bepress taxonomy. Verify with Product on this, though.

Reply:
That's true, but the more technically astute thing to would be to define size on the FE with `page[size]=x`. I will verify this them, because I don't think the FE will know immediately if it's getting custom highlighted or top-level.